app/realm/models/index.js

```diff
@@ -12,6 +13,7 @@ import User from './user';
 export default [
     Channel,
     ChannelMember,
+    ClientUpgrade,
```

Comment:
See comment on the model

Reply:
Here in the index.js we are pulling all the models together. From looking at the code I believe the main benefit of this is in how we instantiate Realm as it takes this index and opens the DB and creates the schema from this.

Reply:
I understood what is doing, just wanted to bring your attention as the model may not even be needed.

Reply:
Correct, if we are not going to persist the data, since it doesn't seem to be needed, then this is not needed.